ixwebsocket/IXWebSocketTransport.h

```diff
@@ -68,7 +68,8 @@ namespace ix
         ~WebSocketTransport();
 
         void configure(const WebSocketPerMessageDeflateOptions& perMessageDeflateOptions,
```

Comment:
the indent nerd in me sees extra spaces here.

Reply:
😅